sklearn/ensemble/iforest.py

```diff
@@ -330,9 +330,10 @@ def decision_function(self, X):
 
         Parameters
         ----------
-        X : {array-like, sparse matrix}, shape (n_samples, n_features)
-            The training input samples. Sparse matrices are accepted only if
-            they are supported by the base estimator.
+        X : array-like or sparse matrix, shape (n_samples, n_features)
```

Comment:
why do you need to change the docstring here?

Reply:
because it was wrong (there is no choice in the base estimator)